### PR TITLE
grafana 7.2.1

### PR DESCRIPTION
**Pull request by bashofmann**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]

**Comment by chenrui333** [comment]
Looks like gopath can be removed as grafana is using go modules for the build.

**Comment by bashofmann** [comment]
I tried to remove the gopath, installation still works, test works and I can start grafana locally. Did I miss something?

File: Formula/grafana.rb [hunk 1]
```diff
@@ -25,26 +25,20 @@ class Grafana < Formula
...
-      prefix.install_metafiles
-    end
+    bin.install "bin/darwin-amd64/grafana-cli"
+    bin.install "bin/darwin-amd64/grafana-server"
+    (etc/"grafana").mkpath
+    cp("conf/sample.ini", "conf/grafana.ini.example")
+    etc.install "conf/sample.ini" => "grafana/grafana.ini"
+    etc.install "conf/grafana.ini.example" => "grafana/grafana.ini.example"
+    pkgshare.install "conf", "public", "tools"
+    prefix.install_metafiles
```
Comment (SMillerDev) [inline Formula/grafana.rb:41]: This shouldn't be needed now.
```suggestion
```

**Comment by chenrui333** [comment]
> something

nope, that is the way to go. :)

**Review by dawidd6 — APPROVED** [review]

**Pull request closed without merging**